src/components/SignOptions.jsx

```diff
@@ -0,0 +1,27 @@
+import ButtonAuth from "./ButtonAuth";
+import OrDivider from "./OrDivider";
+import GoogleIcon from "../../public/GoogleIcon.png";
```

Comment:
Good work! As @Adeeb2022 said, using the `~/` is preferable to using something like `../../`.